scripts/cnode-helper-scripts/mithril-client.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2086
+#shellcheck source=/dev/null
+
+. "$(dirname $0)"/env offline
+
+######################################
+# User Variables - Change as desired #
+# Common variables set in env file   #
+######################################
+
+#MITHRILBIN="${HOME}"/.local/bin/mithril-client # Path for mithril-client binary, if not in $PATH
+
+######################################
+# Do NOT modify code below           #
+######################################
+
+#####################
+# Functions         #
+#####################
+
+usage() {
+  cat <<-EOF
+		
+		Usage: $(basename "$0") [-d] [-u]
+		
+		Cardano Mithril client wrapper script !!
+		-d    Download latest Mithril snapshot
+		-u    Update mithril environment file
+		-h    Show this help text
+		
+		EOF
+}
+
+
+generate_environment_file() {
+  export ERA_READER_ADDRESS=https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/${RELEASE}-${NETWORK_NAME,,}/era.addr
+  export ERA_READER_VKEY=https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/${RELEASE}-${NETWORK_NAME,,}/era.vkey
+  if [[ -n "${POOL_NAME}" ]] && [[ "${POOL_NAME}" != "CHANGE_ME" ]]; then
+    bash -c "cat <<-'EOF' > ${CNODE_HOME}/mithril/mithril.env
```

Comment:
Assignment closer to usage.


```suggestion
generate_environment_file() {
  if [[ -n "${POOL_NAME}" ]] && [[ "${POOL_NAME}" != "CHANGE_ME" ]]; then
    export ERA_READER_ADDRESS=https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/${RELEASE}-${NETWORK_NAME,,}/era.addr
    export ERA_READER_VKEY=https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/${RELEASE}-${NETWORK_NAME,,}/era.vkey
    bash -c "cat <<-'EOF' > ${CNODE_HOME}/mithril/mithril.env
```